frontend/src/components/fetchSpreadsheet.ts

```diff
@@ -0,0 +1,13 @@
+import Papa from 'papaparse';
```

Comment:
not a blocker- I'd move logic into a folder under `/src/sheets/client.ts` or something like that. `components/` seems like a place for `*.vue` files.